tests/modules/test_rnn.py

```diff
@@ -533,17 +527,17 @@ def hook(model, input_, counter):
             counters[str(name)] = counter
             quantizer.register_forward_pre_hook(partial(hook, counter=counter))
         dummy_forward_fn(model)
-        assert model.get_graph().get_nodes_count() == 232  # NB: may always fail in debug due to superfluous 'cat' nodes
-        assert len(counters) == 57
+        assert model.get_graph().get_nodes_count() == 318  # NB: may always fail in debug due to superfluous 'cat' nodes
+        assert len(counters) == 143
         for name, counter in counters.items():
             if 'cell' in name or "LSTMCellForwardNNCF" in name:
                 assert counter.count == sequence_size, name
             else:
                 assert counter.count == 1, name
         new_seq_len = int(sequence_size / 2)
         dummy_forward_fn(model, new_seq_len)
-        assert model.get_graph().get_nodes_count() == 232  # NB: may always fail in debug due to superfluous 'cat' nodes
-        assert len(counters) == 57
+        assert model.get_graph().get_nodes_count() == 318  # NB: may always fail in debug due to superfluous 'cat' nodes
+        assert len(counters) == 143
```

Comment:
it seems, something is broken in RNN

Reply:
Quantizable subgraph patterns had been applied in an incorrect way before - they matched patterns that had output or input edges in the middle of them. Now the patterns are applied correctly, and they no longer match the portions in RNNs they used to match before. It is impossible to specify the previous patterns using current subgraph quantization syntax.